ext/mbstring/tests/zend_multibyte-15.phpt

```diff
@@ -13,6 +13,5 @@ declare(encoding="UTF-8") {
     echo "ok\n";
 }
 ?>
---EXPECT--
-ok
-ok
+--EXPECTF--
+Fatal error: Encoding declaration pragma must not use block mode in %s on line %d
```

Comment:
Makes no sense to test block mode thrice. Just change the test to not use block mode (or drop it if redundant).

Reply:
Yeah, right! This was just to get CI green quickly. I'll update/fix this soonish.